mantle/platform/api/aws/images.go

```diff
@@ -331,14 +332,29 @@ func (a *API) CreateImportRole(bucket string) error {
 
 func (a *API) CreateHVMImage(snapshotID string, diskSizeGiB uint, name string, description string, architecture string) (string, error) {
 	var awsArch string
+	var bootmode string
 	if architecture == "" {
 		architecture = runtime.GOARCH
 	}
+	volumeType, set := os.LookupEnv("aws_volume_type")
+	if !set {
+		volumeType = "gp3"
+	}
+	imdsSupport, set := os.LookupEnv("aws_imds_support")
+	if !set {
+		imdsSupport = ""
+	}
+	bootmode, set = os.LookupEnv("aws_boot_mode")
```

Comment:
Assuming we do https://github.com/coreos/coreos-assembler/pull/3586#discussion_r1321964562 WDYT about moving the `!set` logic up here and renaming this variable to signify it's specific to x86_64?

```suggestion
	bootmodex86, set = os.LookupEnv("aws_boot_mode_x86_64")
	if !set {
	    bootmodex86 = "uefi-preferred"
	}
```